src/eko/anomalous_dimensions/as1aem1.py

```diff
@@ -370,43 +359,16 @@ def gamma_nsm(N, nf, sx):
             + N
             * (
                 96
-                - 3
-                * N
-                * (8 + 64 * g3n * N * (1 + N) ** 3 + 3 * N * (3 + N) * (3 + N**2))
-                + 8 * N * (1 + N) ** 2 * np.pi**2
+                - 3 * N * (8 + 3 * N * (3 + N) * (3 + N**2))
+                + 48 * N * (1 + N) ** 2 * zeta2
             )
+            - 96 * N**3 * (1 + N) ** 3 * (g3N + g3Np2)
```

Comment:
Looking at this naively I can see that if you group by g3 they move to the same level of the s-es and hence without any n prefactor ...